include/audio_player.h

```diff
@@ -50,7 +50,9 @@
 #include <stddef.h>
 #include <stdio.h>
 #include "esp_err.h"
-#include "driver/i2s.h"
+#include "freertos/FreeRTOS.h"
+#include "freertos/semphr.h"
```

Comment:
Can we move the semaphore include to the test if it isn’t used here?

Reply:
> 
yeah, right now.